network-api/networkapi/wagtailpages/tests/buyersguide/test_homepage.py

```diff
@@ -56,9 +56,9 @@ def setUp(self):
             buyersguide = BuyersGuidePage()
             buyersguide.title = 'Privacy not included'
             buyersguide.slug = 'privacynotincluded'
+            buyersguide.call_to_action = buyersguide_factories.BuyersGuideCallToActionFactory()
```

Comment:
**Question** Have you considered only adding this to the buyer's guide homepage in the test cases where it is needed?

Reply:
Updated!